src/functions/routers/action_router_test.ts

```diff
@@ -0,0 +1,355 @@
+import { assertEquals, assertExists, assertMatch } from "../../dev_deps.ts";
+import { BlockActionsRouter } from "./action_router.ts";
+import { DefineFunction, Schema } from "../../mod.ts";
+import { DEFAULT_ACTION, SlackActionHandlerTester } from "../tester/mod.ts";
+
+// Test fixtures: a basic function definition and associated block action router to test
+const func = DefineFunction({
+  callback_id: "id",
+  title: "test",
+  source_file: "whatever",
+  input_parameters: {
+    properties: {
+      garbage: { type: Schema.types.string },
+    },
+    required: [],
+  },
+});
+const { createContext } = SlackActionHandlerTester(func);
+const inputs = { garbage: "in, garbage out" };
```

Comment:
I think we should test for the case of accessing inputs from within a handler. In this case, we should have `ctx.inputs.garbage` available to us. This totally works, but figure it's good to add a test and make sure it continues to work 😅